src/plugins/intel_gpu/src/graph/include/primitive_inst.h

```diff
@@ -49,6 +49,9 @@ struct primitive_impl {
     virtual bool is_cpu() const { return true; }
     virtual void init_kernels() = 0;
     virtual std::unique_ptr<primitive_impl> clone() const = 0;
+    virtual std::vector<std::string> get_kernel_ids() {
```

Comment:
I believe this method need to be overridden in custom_primitive as well

Reply:
I added this method to custom_primtive.